src/test/scala/com/databricks/spark/avro/AvroSuite.scala

```diff
@@ -137,9 +140,9 @@ class AvroSuite extends FunSuite {
 
     val tempDir = Files.createTempDir()
     val avroDir = tempDir + "/avro"
-    AvroSaver.save(TestSQLContext.avroFile(testToAvroFile), avroDir)
+    AvroSaver.save(TestSQLContext.avroFile(testFile), avroDir)
```

Comment:
testToAvroFile was there for testing AvroSaver since the very beginning, because testFile had problems with nullables. Everything is OK now, so testFile can be used for both purposes.